### ISSUE TITLE
Autofilter with merged cells is ugly

### ISSUE THREAD

**Issue by mattwTS** [EPPlusSoftware/EPPlus#302]
I'm generating a report that needs a particular heading layout that's easiest to achieve by merging cells. I'd like to be able to autofilter the table with the report data though, and EPPlus doesn't seem to support emitting the XML necessary to get the filtering to look right. Instead I get filter buttons for every column, even when the headings are spanning columns.

In the attached barebones sample [Central Park Temperatures.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5935400/Central.Park.Temperatures.xlsx), if I unzip and pretty-print the worksheet XML, the autofilter looks like this:

```xml
	<autoFilter ref="A3:D10" xr:uid="{3DF68DB6-F298-4906-9950-9025B1E2181F}">
		<filterColumn colId="0" showButton="0"/>
		<filterColumn colId="2" showButton="0"/>
	</autoFilter>
```

I tried the following:
```C#
	var filterColumns = worksheet.AutoFilter.Columns.AddDynamicFilterColumn(0);
	filterColumn.Type = eDynamicFilterType.Null;
	filterColumn.ShowButton = false;
```
But with that, Excel complains about having to repair the sheet, the buttons still show for every column, all the rows are hidden, and the filterColumn elements wind up looking like this (note: no `showButton` attribute, despite the `ShowButton = false`)

```XML
		<filterColumn colId="0">
			<dynamicFilter type="null" />
		</filterColumn>
		<filterColumn colId="2">
			<dynamicFilter type="null" />
		</filterColumn>
```
Using GitHub search on the EPPlus repository for filterColumn seems to show there's no way to have the API generate a `<filterColumn>` element without a child, which is what I apparently need.

For now I believe I can work around by modding the XML by hand, it'd be nice to have more direct support for this scenario though.

**Comment by JanKallman** [comment]
The ShowButton property did not set the attribute correctly. Fix added in latest develop build.
This code should work...
```C#
                ws.Cells["A1:B1"].Merge = true;
                ws.Cells["A1:C3"].AutoFilter = true;
                ws.AutoFilter.Columns.AddValueFilterColumn(0);
                ws.AutoFilter.Columns[0].ShowButton = false;

```

**Comment by mattwTS** [comment]
Looks like it does the job for me, thanks!

**Comment by JanKallman** [comment]
Fixed in 5.5.4
